stock_inventory_line_product_cost/README.rst

```diff
@@ -19,3 +19,4 @@ Contributors
 ------------
 
 * Open Source Integrators <contact@opensourceintegrators.com>
+* Urvisha Desai <udesai@opensourceintegrators.com>
```

Comment:
Do not modify the Readme directly, as it is self-generated, but create the README folder.
SEE: https://github.com/OCA/maintainer-tools/tree/master/template/module/readme

Reply:
@zamberjo That's only the case when readme fragments are available. This is not the case